specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/stable/2021-04-01/externalIdentities.json

```diff
@@ -0,0 +1,1167 @@
+    },
+    "OperationDisplay": {
+      "description": "Operation display payload",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Resource provider of the operation",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Resource of the operation",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Localized friendly name for the operation",
+          "type": "string"
+        },
+        "description": {
+          "description": "Localized friendly description for the operation",
+          "type": "string"
+        }
+      }
+    },
+    "B2CTenantResourceProperties": {
+      "description": "Properties of the Azure AD B2C tenant Azure resource.",
+      "type": "object",
+      "properties": {
+        "billingConfig": {
+          "type": "object",
+          "description": "The billing configuration for the tenant.",
+          "properties": {
+            "billingType": {
+              "type": "string",
+              "description": "The type of billing. Will be MAU for all new customers. If 'Auths', it can be updated to 'MAU'. Cannot be changed if value is 'MAU'. Learn more about Azure AD B2C billing at [aka.ms/b2cBilling](https://aka.ms/b2cbilling).",
+              "enum": [
+                "MAU",
+                "Auths"
+              ],
+              "x-ms-enum": {
+                "name": "BillingType",
+                "modelAsString": false,
+                "values": [
+                  {
+                    "value": "MAU",
+                    "description": "Azure AD B2C usage is billed to a linked Azure subscription and uses a monthly active users (MAU) billing model."
+                  },
+                  {
+                    "value": "Auths",
+                    "description": "Azure AD B2C usage is billed to a linked Azure subscription and uses number of authentications based billing."
+                  }
+                ]
+              }
+            },
+            "effectiveStartDateUtc": {
+              "type": "string",
+              "readOnly": true,
+              "description": "The data from which the billing type took effect"
+            }
+          }
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "An identifier of the Azure AD B2C tenant."
+        }
+      }
+    },
+    "B2CResourceSKU": {
+      "description": "SKU properties of the Azure AD B2C tenant. Learn more about Azure AD B2C billing at [aka.ms/b2cBilling](https://aka.ms/b2cBilling).",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the SKU for the tenant.",
+          "enum": [
+            "Standard",
+            "PremiumP1",
+            "PremiumP2"
+          ],
+          "x-ms-enum": {
+            "name": "B2CResourceSKUName",
+            "modelAsString": false,
```

Comment:
>false [](http://example.com/codeflow?start=29&length=5)

Consider set it to true to allow adding new enum values in the future without a breaking change.

[https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#use-extensible-enums](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#use-extensible-enums)